docker/glibc/Dockerfile

```diff
@@ -1,13 +1,33 @@
-FROM centos:6 as centos-with-vsyscall
+FROM i386/centos:6 as centos-with-vsyscall32
```

Comment:
why centos 6?

Reply:
This is for manylinux2010 - I'm using the same image here as the native manylinux2010 i686 build (i.e. `docker/Dockerfile-i686`).

Reply:
ok got it